lib/src/create_report.dart

```diff
@@ -911,18 +911,14 @@ Future<ReportSection> _nullSafety(String packageDir, Pubspec pubspec) async {
     if (pubspec.sdkConstraintStatus.hasOptedIntoNullSafety) {
       if (nullSafetyTags.contains('is:null-safe')) {
         subsection = _Subsection(
-            'Package and dependencies are fully migrated to null-safety, '
-            'and will be awarded additional points in a planned future revision '
-            'of the pub.dev points model.',
+            'Package and dependencies are fully migrated to null-safety!',
```

Comment:
No hyphen in `null-safety` (generally, not just here)

Reply:
Thanks! Done